### PR TITLE
Refactor buttons

### PR DESCRIPTION
**Pull request by wellhairy**

### CODE REVIEW

File: assets/scss/partials/campaign/_block-panel.scss [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+  }
 
-//   .button, .button a {
-//     text-decoration: none;
-//   }
-// }
+  .button, .button a {
+    text-decoration: none;
+  }
+}
```
Comment (wellhairy) [inline assets/scss/partials/campaign/_block-panel.scss]: It looks like I need to remove this code all together

File: assets/scss/partials/campaign/_block-panel.scss [hunk 2]
```diff
@@ -137,12 +137,12 @@
...
-      //   background-color: transparent;
-      //   color: $deep-sea-teal;
-      //   text-decloration: none;
-      // }
+      * {
+        margin-top: 0px;
+        background-color: transparent;
+        color: $deep-sea-teal;
+        text-decloration: none;
+      }
```
Comment (wellhairy) [inline assets/scss/partials/campaign/_block-panel.scss]: Same here; remove

File: assets/scss/partials/campaign/_block-panel.scss [hunk 3]
```diff
@@ -183,7 +183,7 @@
               color: #fff;
               text-decoration: none;
           }
-          a:link, a:visited {
+          a:not(.new-button), a:not(.new-button):visited {
```
Comment (wellhairy) [inline assets/scss/partials/campaign/_block-panel.scss]: This needs to reference the class of buttons

File: assets/scss/partials/structure/_typography.scss [hunk 4]
```diff
@@ -87,13 +87,13 @@ p:empty {
...
 
-a:link,
+a,
 a:active,
 a:visited {
   text-decoration: none;
 }
 
-a:link,
+a,
```
Comment (wellhairy) [inline assets/scss/partials/structure/_typography.scss]: This will need testing against

File: assets/scss/partials/structure/_buttons.scss [hunk 5]
```diff
@@ -1,271 +1,141 @@
...
-      left: 8px;
+      content: '';
+      display: inline-block;
+      width: 1em;
+      height: 1em;
+      position: absolute;
+      top: 50%; // 13px;
+      right: 0.8em; // 14px;
+      transform: translateY(-50%);
+      background: url('../images/icons/arrow-medium-white-right.svg') 100% 50% no-repeat;
```
Comment (LidorLapid) [inline assets/scss/partials/structure/_buttons.scss]: Please could we incorporate this into the stylesheet?

File: assets/scss/partials/structure/_buttons.scss [hunk 6]
```diff
@@ -1,271 +1,141 @@
...
-    right: 20px;
+  &--wide {
+    width: 100%;
   }
 
-  &:hover {
+  &--teal-arrow {
     &:after {
-      right: 14px;
+      background: url('../images/icons/arrow-medium-teal-left.svg') 0% 50% no-repeat;
```
Comment (LidorLapid) [inline assets/scss/partials/structure/_buttons.scss]: Please could we incorporate this into the stylesheet?

File: assets/scss/partials/structure/_buttons.scss [hunk 7]
```diff
@@ -1,271 +1,141 @@
...
-
-.button.postfix{
-  padding-left: 22px;
-  padding-right: 22px;
-  &:after{
-    content: "";
-    display: none;
+  &--raspberry-arrow {
+    &:after {
+      background: url('../images/icons/arrow-medium-raspberry-left.svg') 0% 50% no-repeat;
```
Comment (LidorLapid) [inline assets/scss/partials/structure/_buttons.scss]: Please could we incorporate this into the stylesheet?

**Comment by wellhairy** [comment]
closed in favour of #293

**Pull request closed without merging**